core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -106,6 +106,7 @@ var TopicEditorPage = function() {
   var dragAndDrop = async function(fromElement, toElement) {
     await browser.executeScript(dragAndDropScript, fromElement, toElement);
   };
+  var toastSuccessElement = element(by.css('.toast-success'));
```

Comment:
Should we move this to waitFor.ts (or general.ts) file so that we can reuse it in other places? (`waitFor.visibilityOfSuccessToast`)

OR it's fine to leave it as-is?

Reply:
Yeah let's add it to a common place to reuse.